plugins/cost-insights/src/components/BarChart/BarChartTick.tsx

```diff
@@ -35,11 +36,18 @@ export const BarChartTick = ({
   width,
   payload,
   visibleTicksCount,
+  details,
 }: BarChartTickProps) => {
   const gutterWidth = 5;
   const labelWidth = width / visibleTicksCount - gutterWidth * 2;
   return (
-    <BarChartLabel x={x} y={y} height={height} width={labelWidth}>
+    <BarChartLabel
+      x={x}
+      y={y}
+      height={height}
+      width={labelWidth}
+      details={details}
```

Comment:
In our prior design, the details button would appear below each active entity (an entity displayed in a bar chart that the user is hovering over). We pivoted from this design to a clickable entity + tooltip.

But it's likely that we'll want to render a button below an active entity for some other purpose in the near future. 

Since this component was already built out and the details prop as already optional, we can preserve a bit of the work thats already been done and implement this feature later.